### DISCUSSION TITLE
addRoute方法无法添加子路由

### DISCUSSION THREAD

**Discussion by conanliuhuan** [vuejs/router#2032] [Help and Questions]
addRoute(parentName, route)
官方文档表示，只要第一个参数写了parentName，那么就应该添加为parentName路由的子路由。但事实上却并非如此，仍然只是添加到根记录，并不是添加到父记录的children里面。
这是我理解有误吗？还是BUG哦？
如果我想要添加到某个父记录的children里面应该怎么做？

**Comment by luqiudi** [comment]
你确定下，你加的是name 不是path？

**Reply by gswy** [reply]
我也遇到了一样的问题。填写了上级路由的name，但是始终添加到根，而不是上级目录下。

**Reply by zp-Han** [reply]
有解决吗？我也是这样👀
